tests/lib/bits.test.ts

```diff
@@ -115,11 +115,13 @@ describe('Bits', () => {
 			const value =
 				GuildSystemChannelFlags.SuppressJoinNotifications |
 				GuildSystemChannelFlags.SuppressPremiumSubscriptions |
-				GuildSystemChannelFlags.SuppressGuildReminderNotifications;
+				GuildSystemChannelFlags.SuppressGuildReminderNotifications |
+				8; // TODO: GuildSystemChannelFlags.SuppressJoinNotificationReplies
```

Comment:
This hack needed to be done because https://github.com/discordjs/discord-api-types/pull/222 is unreleased.

Reply:
now it is

Reply:
But we need discord.js with the update, otherwise there are many conflicts :angery: